engine/apps/slack/views.py

```diff
@@ -270,6 +270,12 @@ def post(self, request):
                 # Open pop-up to inform user why OnCall bot doesn't work if any action was triggered
                 self._open_warning_window_if_needed(payload, slack_team_identity, warning_text)
                 return Response(status=200)
+        elif not slack_user_identity.users.exists():
+            # Means that slack_user_identity doesn't have any connected user
+            warning_text = "Permission denied. Please connect your Slack account to OnCall."
+            # Open pop-up to inform user why OnCall bot doesn't work if any action was triggered
+            self._open_warning_window_if_needed(payload, slack_team_identity, warning_text)
```

Comment:
Probably we can use `_open_warning_for_unconnected_user` here?

Reply:
Makes sense